### DISCUSSION TITLE
I have "nerdctl" and "docker" installed, but docker commands don't work

### DISCUSSION THREAD

**Discussion by navynight** [rancher-sandbox/rancher-desktop#1511] [Q&A]
I examined: \AppData\Local\Programs\Rancher Desktop\resources\resources\win32\bin

I see several applications such as: kubectl, nerdctl, **docker**, etc.

Does this mean that when I have Rancher Desktop running, I should be able to run "docker" commands on the command line, similar to "nerdctl" commands?

I tried one docker command, but it failed:
`docker volume ls`

Output: 
request returned Bad Gateway for API route and version
http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/volumes, check if the server supports the requested API version

With **nerdctl**, it works.

Command: 
`nerdctl volume ls`

Output:
VOLUME NAME    DIRECTORY
my_data    /var/lib/nerdctl/.../volumes/default/my_data/_data

Could this be a bug?

**Comment by jandubois** [comment]
> Does this mean that when I have Rancher Desktop running, I should be able to run "docker" commands on the command line, similar to "nerdctl" commands?

In the "Kubernetes settings" page you can select the container runtime: either `containerd` or `dockerd/moby`. You use `nerdctl` with `containerd` and the `docker` cli with `dockerd/moby`. You cannot mix'n'match; each CLI only works for the corresponding runtime.

**Reply by navynight** [reply]
Thank you for your answer!

When I run the dockerd/moby container runtime and the command `docker ps` I get the following output:

error during connect: This error may indicate that the docker daemon is not running.: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/containers/json: open //./pipe/docker_engine: The system cannot find the file specified.

Is this a bug?

**Reply by tripaak** [reply]
Hello @navynight Have you found the solution ?
I am in the same situation.

**Reply by varunmbhat84** [reply]
I also have same Issue? can you let me know the solution for this please! @navynight @tripaak @ryanfb @jandubois @flavio

**Reply by jandubois** [reply]
Please create a GitHub issue with all the details of your setup and attach a zipfile of your logs.
